static/js/components/AutoComplete.js

```diff
@@ -393,7 +405,8 @@ class AutoComplete extends Component {
       triggerUpdateOnFocus, // eslint-disable-line no-unused-vars
       openOnFocus, // eslint-disable-line no-unused-vars
       maxSearchResults,
-      dataSource,
+      dataSource, // eslint-disable-line no-unused-vars
+      menuHeight,
```

Comment:
seems odd that `menuHeight` wouldn't be part of `menuStyle`. is there a reason for this? either way, this should be moved up to be proximal to `menuStyle` and `menuProps` in this list